packages/api/src/api/contributions.ts

```diff
@@ -0,0 +1,44 @@
+import { Router } from 'express';
+import { Contribution } from '../entities/Contribution';
+import { User } from '../entities/User';
+import logger from '../logger';
+
+export const contributions = Router();
+
+contributions.get('', async (req, res) => {
+  if (!req.user) {
+    res.sendStatus(401);
+    return;
+  }
+
+  const { userId } = req.query;
+
+  if (!userId) {
+    res.status(400).send('No user id was given.');
+    return;
+  }
+
+  if (Number.isNaN(Number(userId))) {
+    res.status(400).send(`"${userId}" is not a valid id, it must be a number.`);
+    return;
+  }
+
+  try {
+    const queriedUser = await req.entityManager.findOne(User, { id: userId as string });
```

Comment:
After making the entity change below, you can use the [`populate` parameter](https://mikro-orm.io/docs/entity-manager#fetching-entities-with-entitymanager) here to get all the `user`'s contributions

Reply:
You should be able to do it all on one line 👍🏾 
```suggestion
    const queriedUser = await req.entityManager.findOne(User, { id: userId as string }, ['contributionList']);
```